drivers/sx127x/Makefile.dep

```diff
@@ -3,7 +3,9 @@ FEATURES_REQUIRED += periph_gpio_irq
 FEATURES_REQUIRED += periph_spi
 FEATURES_OPTIONAL += periph_spi_gpio_mode
 USEMODULE += iolist
-USEMODULE += xtimer
+USEMODULE += ztimer_usec
+USEMODULE += ztimer_msec
+USEMODULE += ztimer_periph_rtt
```

Comment:
This change will require the `rtt` feature for `pkg_semtech_lorawan` now, are ok with this? you could have something like this instead:

```
  FEATURES_OPTIONAL += periph_rtt
  ifneq (,$(filter periph_rtt,$(FEATURES_USED)))
    USEMODULE += ztimer_periph_rtt
  endif
```

Reply:
Addressed